### PR TITLE
Feature/created new crud handler

### PR DESCRIPTION
**Pull request by GabrielPedrotti**
Created a file that concetrates all the requisitions through serverless and then will executre especific functions depending on wich rote and method selected

### CODE REVIEW

**Review by Viserion77 — CHANGES_REQUESTED** [review]
Esta faltando um eslint ai

File: src/lambda/crudHandler.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { lambdaGetParameters } from './lambdaGetParameters'
+import { APIGatewayEvent } from 'aws-lambda'
+import { error } from '../error'
+import { getDefaultResponse, HttpNames } from '../http'
+
+export const lambdaCrudHandler = (event: APIGatewayEvent): object => {
```
Comment (th3rius) [inline src/lambda/crudHandler.ts:6]: ´object´ isn't a very useful type in here. I'd suggest letting TypeScript infer the return type of those functions (and disabling ´@typescript-eslint/explicit-function-return-type´ for the file).

Comment (Viserion77) [reply src/lambda/crudHandler.ts:6]: let's talk about it in the chatper

Comment (GabrielPedrotti) [reply src/lambda/crudHandler.ts:6]: so I just leave it the way it was @Viserion77 ?

Comment (th3rius) [reply src/lambda/crudHandler.ts:6]: @Viserion77 and I agreed to leave it like that. We can type this later.

**Review by th3rius — APPROVED** [review]

**Review by th3rius — APPROVED** [review]

**Review by Viserion77 — APPROVED** [review]
LGTM

**Review by marcosalves-bitg — APPROVED** [review]
LGTM

**Pull request merged**